From 471ecfbe03cabe1287351b7c16823ddd94bcf998 Mon Sep 17 00:00:00 2001
From: "Xiaokang, Qin" <xiaokang.qin@intel.com>
Date: Sun, 21 Oct 2012 02:48:43 +0800
Subject: [PATCH] Add a flag control that PRIVATE libm can be chosen

Add a flag control that PRIVATE libm can be chosen

Change-Id: Ieb333c5b8b58e26b5da4f5b40d22a3fc13eb5cd5
Category: device enablement
Domain: AOSP-bionic
Origin: internal
Signed-off-by: Xiaokang, Qin <xiaokang.qin@intel.com>
Reviewed-on: http://android.intel.com:8080/74475
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>

diff --git a/libm/Android.mk b/libm/Android.mk
index b6ccc4c..9b0a09f 100644
--- a/libm/Android.mk
+++ b/libm/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(USE_PRIVATE_LIBM),true)
 LOCAL_PATH:= $(call my-dir)
 
 # TODO: these come from from upstream's libc, not libm!
@@ -274,3 +275,4 @@ LOCAL_ADDITIONAL_DEPENDENCIES := $(LOCAL_PATH)/Android.mk
 LOCAL_SYSTEM_SHARED_LIBRARIES := libc
 LOCAL_WHOLE_STATIC_LIBRARIES := libm
 include $(BUILD_SHARED_LIBRARY)
+endif
